### PR TITLE
NG-5118 Update Update-Java.Bash

### PR DESCRIPTION
**Pull request by tucker19**
Updated Update-Java.Bash with latest Amazon Corretto 8 version

### CODE REVIEW

File: profile-seed/.scripts/update-java.bash [hunk 1]
```diff
@@ -127,11 +127,10 @@ if [ "$SET_UP_PROFILE" = true ];
...
     cat >> ~/.profile <<-EOF
-
-	export JAVA_HOME=$JDK_LOCATION
-	export JRE_HOME=$JDK_LOCATION/Contents/Home/jre
-	export PATH=$PATH:$JDK_LOCATION/Contents/Home/bin:$JDK_LOCATION/Contents/Home/jre/bin
-	EOF
+    export JAVA_HOME=$JDK_LOCATION
+    export JRE_HOME=$JDK_LOCATION/Contents/Home/jre
+    export PATH=$PATH:$JDK_LOCATION/Contents/Home/bin:$JDK_LOCATION/Contents/Home/jre/bin
+    EOF
```
Comment (jwilson194) [inline profile-seed/.scripts/update-java.bash]: For me at least I had to delete the tab in front of `EOF` (moved all the way to the left) to get this to run correctly using the `--profile` option

**Comment by jwilson194** [comment]
This part looks good to me and the update seems to have gone on without issue, not sure at what point we will call this one good to be merged/updated on prod though

File: profile-seed/.scripts/update-java.bash [hunk 2]
```diff
@@ -45,12 +45,12 @@ if [ $# -ne 0 ]; then
     OPTS='getopt -q -o '' -l profile,help -- "$@"'
     while true; do
         case "$1" in
-            --profile ) SET_UP_PROFILE=true; shift; break ;;
```
Comment (jwilson194) [inline profile-seed/.scripts/update-java.bash]: This break was needed when the `--profile` option was used.

Comment (tucker19) [reply profile-seed/.scripts/update-java.bash]: Added back in

**Pull request merged**